src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -70,10 +73,60 @@ const config = ConfigStore.getConfig();
 
 const tracesSampleRate = config ? config.apmSampling : 0;
 
+const appRoutes = Router.createRoutes(routes());
+const createLocation = createMemoryHistory().createLocation;
+
 Sentry.init({
   ...window.__SENTRY__OPTIONS,
   integrations: getSentryIntegrations(),
   tracesSampleRate,
+  async beforeSend(event) {
```

Comment:
Can we maybe move this in a helper function to not have so much code in `bootstrap.tsx`?

```
import { fetchTransaction } from 'app/utils/apm';
...
beforeSend: fetchTransactions
...
```

Reply:
done in https://github.com/getsentry/sentry/pull/18822/commits/7a5ba50c91352ded7b64dc95c2f759781eaaa065